python/datasources/acs_health_insurance.py

```diff
@@ -0,0 +1,657 @@
+import pandas as pd
+
+import requests
+import json
+from datasources.data_source import DataSource
+from ingestion import url_file_to_gcs, gcs_to_bq_util
+
+from ingestion.standardized_columns import (STATE_FIPS_COL, COUNTY_FIPS_COL,
+                                            STATE_NAME_COL, COUNTY_NAME_COL,
+                                            AGE_COL, SEX_COL,
+                                            RACE_COL, WITH_HEALTH_INSURANCE_COL,
+                                            WITHOUT_HEALTH_INSURANCE_COL,
+                                            TOTAL_HEALTH_INSURANCE_COL)
+# TODO pass this in from message data.
+BASE_ACS_URL = "https://api.census.gov/data/2019/acs/acs5"
+
+
+class HealthInsurancePopulation:
```

Comment:
Intellisense mostly.  It acts as an enum and you can goto defintion of the variable rather than if you did HEALTH_INSURANCE_POPULATION["WITH" you could only go to the dict defintion.  This doesnt scale well.  Also I think the classes look much cleaner.

Reply:
Fair enough; in that case, do you want to explicitly subclass Enum for these so it's clearer?

Reply:
Subclassing Enum in python is wierd